adapters/adoppler/adopplertest/exemplary/multibid.json

```diff
@@ -1,60 +1,50 @@
-                                          "seatbid": [{"bid": [{"id": "req1-imp1-bid1",
-                                                                "impid": "imp1",
-                                                                "price": 0.12,
-                                                                "adm": "<b>a banner</b>"}]}],
-                                          "cur": "USD"}}},
-               {"expectedRequest": {"uri": "http://adoppler.com/processHeaderBid/unit2",
-                                    "body": {"id": "req1-unit2",
-                                        "imp": [{"id": "imp2",
-                                                 "video": {"minduration": 120,
-                                                           "mimes": ["video/mp4"]},
-                                                 "ext": {"bidder": {"adunit": "unit2"}}}]}},
-                "mockResponse": {"status": 200,
-                                 "body": {"id": "req1-imp2-resp2",
-                                          "seatbid": [{"bid": [{"id": "req1-imp2-bid1",
-                                                                "impid": "imp2",
-                                                                "price": 0.24,
-                                                                "adm": "<VAST />",
-                                                                "cat": ["IAB1", "IAB2"],
-                                                                "ext": {"ads": {"video": {"duration": 121}}}}]}],
-                                          "cur": "USD"}}},
-               {"expectedRequest": {"uri": "http://adoppler.com/processHeaderBid/unit3",
-                                    "body": {"id": "req1-unit3",
-                                        "imp": [{"id": "imp3",
-                                                 "native": {"request": "{}"},
-                                                 "ext": {"bidder": {"adunit": "unit3"}}}]}},
-                "mockResponse": {"status": 204,
-                                 "body": ""}}],
- "expectedBidResponses": [{"currency": "USD",
-                           "bids": [{"bid": {"id": "req1-imp1-bid1",
-                                             "impid": "imp1",
-                                             "price": 0.12,
-                                             "adm": "<b>a banner</b>"},
-                                     "type": "banner"}]},
-                          {"currency": "USD",
-                           "bids": [{"bid": {"id": "req1-imp2-bid1",
-                                             "impid": "imp2",
-                                             "price": 0.24,
-                                             "adm": "<VAST />",
-                                             "cat": ["IAB1", "IAB2"],
-                                             "ext": {"ads": {"video": {"duration": 121}}}},
-                                     "type": "video"}]}]}
+{
```

Comment:
Thank you for formatting the json tests. Much easier to read now. Hence this question.. what part of this test case is "multi"? This looks like a no-bid response.

Reply:
Good catch, thanks. I've accidentally replaced this file with wrong content during formatting. It verifies multi-impression scenario. Will fix and rename it to `multiimp.json` to.